src/m365/todo/commands/task/task-list.ts

```diff
@@ -0,0 +1,120 @@
+import * as chalk from 'chalk';
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import commands from '../../commands';
+import { ToDoTask } from '../../ToDoTask';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  listName?: string;
+  listId?: string;
+}
+
+class ToDoTaskListCommand extends GraphItemsListCommand<ToDoTask> {
```

Comment:
Can we please rename it to `TodoTaskListCommand` to keep it consistent with the other command class name. D need not be capital letter although `ToDo` makes more sense, but looking at our older commands this may be the consistent approach. Thoughts? @pnp/cli-for-microsoft-365-maintainers

Reply:
Let's use `TodoTasksListCommand` like you suggested and stay consistent. Our commands are after all named `todo ...` rather than `to do ...` 🙂